website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -83,7 +83,15 @@ Finally, set the "Private Key" and "Private Key Passphrase" fields in the "Edit
 Credentials" page to finish configuring dbt Cloud to authenticate with Snowflake
 using a key pair.
 
-**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud -- you **must** add the passphrase that will be used to decrypt the and will receive an error if the `PRIVATE KEY PASSPHRASE` field is empty.
+**Note:** At this time ONLY Encrypted Private Keys are supported by dbt Cloud. In order to successfully fill in the Private Key field, you  **must** include the commented lines below when you add the passphrase. The passphrase you add, will be used to decrypt the and leaving the `PRIVATE KEY PASSPHRASE` field empty will will return an error.
```

Comment:
The 2nd sentence here might be incomplete - "the passphrase you add, will be used to decrypt the {{ missing word, key? }} and".

Also looks like maybe that should be a line break from the "Note" about the commented lines.

Otherwise looks good to go!

Reply:
hi @krevitt that's a great callout and thank you so much for those suggestions! I've made those changes and also removed the duplicate 'will' at the end as well. hope this is good to go now and appreciate the help!